### PR TITLE
ch -front-tests

### PR DESCRIPTION
**Pull request by remyroo**
#### What does this PR do
- Add test for team access
- Add test for the /integrations_add/ url edge case
- Add test for pushover validation
#### Any background context you want to provide

*Appears to be a repetition of the team access test on _test_add_check.py_
*Corrected the typo on the file _test_add_pushbover.py_ to _test_add_pushover.py_ but the original remains on GitHub

### CODE REVIEW

File: hc/front/tests/test_add_pushover.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        params = "pushover_user_key=a&nonce=n&prio=0"
+        r = self.client.get("/integrations/add_pushover/?%s" % params)
+        assert r.status_code == 302
+
+        channels = list(Channel.objects.all())
+        assert len(channels) == 1
```
Comment (mfalade) [inline hc/front/tests/test_add_pushover.py]: Use

``` python
self.assertEqual(len(channels), 1)
```

instead. Also, replace all instances of `assert` to `self.assert{Whatever}`

Comment (arnoldokoth) [reply hc/front/tests/test_add_pushover.py]: @mfalade @andela-rwachira  That particular test was already part of the code base. But I think `assert r.status_code == 302` is specific to pytest and `self.assert{Whatever}` is specific to unittest? Are there any particular advantages of one over the other and is it cool to mix it up?

Comment (mfalade) [reply hc/front/tests/test_add_pushover.py]: Pytest uses just plain `assert` statements.. but `assert` statements are not limited to pytests only. You can use them within your code as needed.

AFAIK, there isn't any particular advantage of using one over another. However, it is important to stick to a consistent style within scripts.

In our case, we are using self.assertStuff() for all tests (and 👎 , it is not cool to mix it up).

Comment (arnoldokoth) [reply hc/front/tests/test_add_pushover.py]: Took a minute to process the meaning of AFAIK. Lol! Thanks and noted.

**Comment by remyroo** [comment]
Hey @mfalade @andela-aokoth, I've updated all asserts in front/tests to self.assert. According to Travis they are now all passing. I'm not able to run the tests locally though so I'm not sure if it's ok for you to merge yet.
Kindly say yes @mfalade so we can check this off Pivotal, we need at least one win this week :)
